converters/fromZigbee.js

```diff
@@ -5561,6 +5561,38 @@ const converters = {
             return payload;
         },
     },
+    tuya_brightness: {
```

Comment:
Can be removed, duplicate of https://github.com/Koenkk/zigbee-herdsman-converters/blob/9025c2d69ad945404263340e3e842b3d071048e0/converters/fromZigbee.js#L770